vspec2csv.py

```diff
@@ -57,10 +57,10 @@ def print_csv_content(file, tree):
         if tree_node.instances:
             for instance in tree_node.instances:
                 file.write(format_csv_line(
-                    tree_node.qualified_name('.'),tree_node.type.value,data_type_str,tree_node.deprecation,"true",unit_str,tree_node.min,tree_node.max,tree_node.description,tree_node.enum,tree_node.uuid,instance))
+                    tree_node.qualified_name('.'),tree_node.type.value,data_type_str,tree_node.deprecation,"true",unit_str,tree_node.min,tree_node.max,tree_node.description,tree_node.comment,tree_node.enum,tree_node.uuid,instance))
```

Comment:
Any opinion on where to add comment in *.csv? As of now this code add comment after description which could be logically correct, but it could cause backward compatibility issues if someone use *.csv and rely on absolute position for "enum"/"id". As ana lternative we could add "comment" last.

(By the way - I assume the "instance" code here is obsolete - we have already expanded instanced before we print *.csv, so this line will never be executed)